test/unit/mongo/MongoTripodConfigTest.php

```diff
@@ -56,30 +56,40 @@ public function testNamespaces()
     public function testTConfig()
     {
         $config = MongoTripodConfig::getInstance();
+        $cfg = MongoTripodConfig::getConfig();
         $tConfig = $config->getTransactionLogConfig();
-        $this->assertEquals('testing',$tConfig['database']);
+        $this->assertEquals('tripod_php_testing',$tConfig['database']);
         $this->assertEquals('transaction_log',$tConfig['collection']);
-        $this->assertEquals('mongodb://localhost',$config->getTransactionLogConnStr());
+        $this->assertEquals($cfg['data_sources'][$cfg['transaction_log']['data_source']]['connection'],$config->getTransactionLogConnStr());
     }
 
     public function testTConfigRepSetConnStr()
     {
         $config=array();
+        $config["data_sources"] = array(
```

Comment:
There is a lot of repeated config creation here - maybe a bit out of scope for this particular story to refactor all of it, but worth thinking about having something to do this for you rather than hand craft it for every test.